internal/impl/kube.go

```diff
@@ -94,20 +94,29 @@ type ListenerOptions struct {
 // KubeConfig stores the configuration information for one execution of a
 // Service Weaver application deployed using the Kube deployer.
 type KubeConfig struct {
-	// Image is the name of the container image that "weaver kube deploy"
-	// builds and uploads. For example, if Image is "docker.io/alanturing/foo",
-	// then "weaver kube deploy" will build a container called
-	// "docker.io/alanturing/foo" and upload it to Docker Hub.
+	// Tag is the build tag for the application container on the local
+	// machine.
 	//
-	// The format of Image depends on the registry being used. For example:
+	// If empty, the tag defaults to "<app_name>:<app_version>", where
+	// <app_version> is the unique version id of the application deployment.
+	Tag string
```

Comment:
Should we call this LocalTag given that we can have a LocalTag and a RepoTag?

Reply:
Good point, done.